packages/web3-utils/src/hash.ts

```diff
@@ -57,7 +57,25 @@ export const sha3Raw = (data: Bytes): string => {
 	return hash;
 };
 
-export { keccak256 };
+/**
+ * A wrapper for ethereum-cryptography/keccak256 to allow hashing a `string` and a `bigint` in addition to `UInt8Array`
+ */
+export const keccak256Wrapper = (
+	data: Uint8Array | ReadonlyArray<number> | string | bigint,
```

Comment:
Please include the `Buffer` as well. 
```suggestion
	data: Bytes | Numbers | string,
```

Reply:
Sorry, this is incompatible with keccak256. In web3.js v1.x, It throws an exception if we pass a number that is provided from type `Numbers` or if we pass `ArrayBuffer` that is provided from type `Bytes`. 
Additionally, the `readonly numbers[]` is supported as I am using `Buffer.from(...)` and so I included it.

Reply:
As per our discussion, it is nice to support Bytes and Numbers, similar to other utility functions. 
So, I modified the logic to include more types as desired. Kindly re-check now.